File: Source/Rio.Models/DataTransferObjects/Parcel/ParcelMonthlyEvapotranspirationOverrideDto.cs
namespace Rio.Models.DataTransferObjects.Parcel
{
    public class ParcelMonthlyEvapotranspirationOverrideDto
    {
        public int ParcelID { get; set; }
        public string ParcelNumber { get; set; }
        public int WaterYear { get; set; }
        public int WaterMonth { get; set; }
        public decimal OverriddenEvapotranspirationRate { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/parcel/parcel-monthly-evapotranspiration-override-dto.ts
export class ParcelMonthlyEvapotranspirationOverrideDto {
    ParcelID: number;
    ParcelNumber: string;
    WaterYear: number;
    WaterMonth: number;
    OverriddenEvapotranspirationRate: number;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}